build_windows.md

```diff
@@ -1,65 +1,97 @@
+    - [Allow local PowerShell scripts execution](#allow-local-powershell-scripts-execution)
+- [Build and test the Podman client for Windows](#build-and-test-the-podman-client-for-windows)
+    - [Build the Podman client](#build-the-podman-client)
+    - [Download gvproxy.exe and win-sshproxy.exe](#download-gvproxyexe-and-win-sshproxyexe)
+    - [Create a configuration file (optional)](#create-a-configuration-file-optional)
+    - [Create and start a podman machine](#create-and-start-a-podman-machine)
+    - [Run a container using podman](#run-a-container-using-podman)
+- [Build and test the Podman Windows installer](#build-and-test-the-podman-windows-installer)
+    - [Build the Windows installer bundle](#build-the-windows-installer-bundle)
+- [Validate changes before submitting a PR](#validate-changes-before-submitting-a-pr)
+
+## Requirements
+
+### OS requirements
+
+Windows OS can behave very differently depending on how it was configured. This documentation assumes that one is using a [Windows 11 development machine](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/) or a configuration close to this one. The Podman Windows client installer bundles several tools, which are unnecessary for Podman builds, but this
+set of packages is well aligned with GitHub's `windows-latest` offerings. Some of the tools will still be missing from this distribution and will have to be manually added after this installation completes.
+
+### Git and go
+
+To build Podman, the [git](https://gitforwindows.org/) and [go](https://go.dev) tools are required. In case they are not yet installed,
+open a Windows Terminal and run the following command (it assumes that [winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/) is installed):
+
+```pwsh
+winget install -e GoLang.Go Git.Git
+```
 
-## OS requirements
+:information_source: A terminal restart is advised for the `PATH` to be reloaded. This can also be manually changed by configuring the `PATH`:
 
-Windows OS can behave very differently depending on how it was configured. This documentation assumes that one is using
-a [Windows 11 development machine](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/) or a
-configuration close to this one. The Podman Windows client installer bundles several tools, which are unnecessary for Podman builds, but this
-set of packages is well aligned with GitHub's `windows-latest` offerings. Some of the tools will still be missing from
-this distribution and will have to be manually added after this installation completes.
+```pwsh
+$env:Path += ";C:\Program Files\Go\bin\;C:\Program Files\Git\cmd\"
+```
 
-## Install Pandoc
+### Pandoc
 
+[Pandoc](https://pandoc.org/) is used to generate Podman documentation. It is required for a full build of the Podman Windows installer but can be ignored to build the Podman Windows client or a minimal build of the Podman Windows Installer.
```

Comment:
Can you clarify the difference between the minimal vs full build of the installer?

Reply:
Ah. Good catch. I forgot to update that. The documentation build (and pandoc) is required to [build the bundle installer](#build-the-installer) but not to [build the standalone `podman.msi` file](#build-and-test-the-standalone-podmanmsi-file). I have updated this section.